istioctl/cmd/istioctl/main.go

```diff
@@ -334,16 +328,12 @@ istioctl get destinationrules
 # Get a specific virtual service named bookinfo
 istioctl get virtualservice bookinfo
 `,
+		Args: cobra.MinimumNArgs(1),
```

Comment:
Could you remove this one because the error message gives some context to the user.